library/src/main/java/com/mux/video/upload/internal/UploadMetrics.kt

```diff
@@ -24,16 +24,6 @@ internal class UploadMetrics private constructor() {
 
   private val logger get() = MuxUploadSdk.logger
 
-
-  private fun formatMilliseconds(ms:Long):String {
```

Comment:
This method wasn't used. It looks like we moved to `SimpleDateFormat`, which is definitely 👍 👍